tools/src/main/java/org/apache/rocketmq/tools/admin/DefaultMQAdminExtImpl.java

```diff
@@ -320,8 +321,18 @@ public ConsumeStats examineConsumeStats(String consumerGroup,
         }
 
         if (result.getOffsetTable().isEmpty()) {
-            throw new MQClientException(ResponseCode.CONSUMER_NOT_ONLINE,
-                "Not found the consumer group consume stats, because return offset table is empty, maybe the consumer not consume any message");
+            ConsumerConnection connection;
+            try {
+                connection = this.examineConsumerConnectionInfo(consumerGroup);
+            } catch (Exception e) {
+                throw new MQClientException(ResponseCode.CONSUMER_NOT_ONLINE,
+                    "Not found the consumer group consume stats, because return offset table is empty, maybe the consumer not online");
+            }
+
+            if (connection.getMessageModel().equals(MessageModel.BROADCASTING)) {
+                throw new MQClientException(ResponseCode.CONSUME_BROADCASTING,
+                    "Not found the consumer group consume stats, because return offset table is empty, the consumer using in broadcast model");
```

Comment:
Hi @zhangjidi2016 ,

Thx for your contribution.

Please rephrase the "the consumer using in broadcast model" into "the consumer is under the broadcast mode". A good exception message would make us more perfect :-)